xcm/src/v3/mod.rs

```diff
@@ -78,6 +79,24 @@ impl<Call> Xcm<Call> {
 		self.0.len()
 	}
 
+	/// Return a reference to the inner value.
+	pub fn inner(&self) -> &Vec<Instruction<Call>> { &self.0 }
+
+	/// Return a mutable reference to the inner value.
+	pub fn inner_mut(&mut self) -> &mut Vec<Instruction<Call>> { &mut self.0 }
```

Comment:
Prefer array references rather than hardcoding a `Vec` to make it more generic:

```suggestion
	pub fn inner(&self) -> &[Instruction<Call>] { &self.0 }

	/// Return a mutable reference to the inner value.
	pub fn inner_mut(&mut self) -> &mut [Instruction<Call>] { &mut self.0 }
```